box2d.qbs

```diff
@@ -0,0 +1,30 @@
+import qbs
+
+Project {
+    name: "box2d";
+
+    DynamicLibrary {
+        name: "box2d"
```

Comment:
Here I wonder why Box2D itself is built as dynamic library. It is only used internally by the QML plugin, so juggling a separate library around seems unnecessary. I would change this to a static library.